docs/release_notes/0.4.0.md

```diff
@@ -1,10 +1,24 @@
 # Release 0.4.0
 
 ## Features
-- ...
+
+Includes all features of the previous version, plus:
+- New experimental feature `generate profile` (#1132)
+- Added installation of Helm/Tiller as part of the `install flux` operation (#1136)
 
 ## Bug Fixes and code improvements
 - perform defaulting and validation in all commands (#1124)
 - improve nodegroup validation to detect duplicate names (#1116)
 - tidy up command usage formatting (#1112)
 - various internal improvements (#1113, #1117, #1118, #1119)
+- remove the ability of skipping tasks (#1101)
+- added tests for experimental features
+- refactored `install flux` experimental feature for testability
+- added common AWS CLI options to `install flux`
+- cmd description spacing fixed
```

Comment:
Please definitely remote this, it's just meaningless to anyone who reads this.

Reply:
Remove what? the 4 lines?